site/_includes/partials/post-headline.njk

```diff
@@ -1,4 +1,4 @@
-<h1 class="type--h2">{{ title }}</h1>
+<h1 class="type--h2">{{ title | md | safe }}</h1>
```

Comment:
Note: This fixes an inconsistency where pages like https://developer.chrome.com/articles/ would already convert Markdown in titles to HTML, but the article detail pages wouldn’t. https://github.com/GoogleChrome/developer.chrome.com/blob/4e0c380351a22dd094435b7c39492257e7dd6d8d/site/_includes/macros/cards/blog-card.njk#L37 This is important for this article since it uses a code span in its title.